hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyInfo.java

```diff
@@ -288,7 +289,17 @@ public Builder setKeyName(String key) {
 
     public Builder setOmKeyLocationInfos(
         List<OmKeyLocationInfoGroup> omKeyLocationInfoList) {
-      this.omKeyLocationInfoGroups = omKeyLocationInfoList;
+      if (omKeyLocationInfoList != null) {
```

Comment:
Why can this be null? Is it straightforward to replace null with `Collections.emptyList()` in caller? Generally good to avoid nulls.

Reply:
This will never be null, but someone can use this API and pass null, to avoid those scenarios added that check.